### PR TITLE
Removed the dependency on TheMovieDb and made it optional

### PR DESCRIPTION
**Pull request by lclc98**
Allowed for moviedb to not be used if the key is not present

### CODE REVIEW

**Comment by ManuGithubSteam** [comment]
Hmm ok, i think MovieDB is a great feat but i see where are you going with this :-)

**Comment by lclc98** [comment]
MovieDB is great, but some people dont like all the details they require

**Comment by ManuGithubSteam** [comment]
Trash mail helps :)

File: grails-app/assets/javascripts/streama/controllers/dash-ctrl.js [hunk 1]
```diff
@@ -3,19 +3,6 @@
...
-				var TheMovieDbAPI = _.find(data, {settingsKey: 'TheMovieDB API key'});
-
-				if (!TheMovieDbAPI.value) {
-					alertify.alert('You need to fill out some required base-settings. You will be redirected to the settings page now.', function () {
-						$state.go('settings.settings');
-					});
-				}
-			});
-		}
-
```
Comment (dularion) [inline grails-app/assets/javascripts/streama/controllers/dash-ctrl.js]: You cant remove this part of the code, its important for the initial setup, but maybe query against another missing setting, like the upload directory?

**Pull request merged**